CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing
+First off, thank you for considering contributing to Feed a Doc. We hope to fill a big void and we need all the help we can get.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+## Issue Tracking
+We do our best to keep issues up-to-date. There are some simple things you can do to help us. 
+
+- If you start working on an issue, claim it by commenting on the issue
+- If you see a ticket has been claimed but there has been no activity for a week, feel free to check with the person that claimed the issue. If you don't get a response, claim the issue.
+
+## Development guidelines
+The goal of this project is to get a good product off the ground quickly as things are rapidly changing. As such, following these guidelines will really help.
+- Consider other developers when adding tools/changes. If the change requires some obscure tooling or changes the default way of doing things, be sure to provide thorough documentation.
```

Comment:
None of these points are strongly held, just a starting point for getting this in place.

Reply:
I agree with them, nice!